zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/ListViewProcessInstanceFromIncidentHandler.java

```diff
@@ -66,25 +72,42 @@ public ProcessInstanceForListViewEntity createNewEntity(final String id) {
   public void updateEntity(
       final Record<IncidentRecordValue> record, final ProcessInstanceForListViewEntity entity) {
 
-    final List<List<Long>> elementInstancePath = record.getValue().getElementInstancePath();
-    final List<Integer> callingElementPath = record.getValue().getCallingElementPath();
+    final IncidentRecordValue value = record.getValue();
+    final List<List<Long>> elementInstancePath = value.getElementInstancePath();
+    final List<Integer> callingElementPath = value.getCallingElementPath();
+    final List<Long> processDefinitionPath = value.getProcessDefinitionPath();
 
     final Long processInstanceKey = Long.valueOf(entity.getId());
 
+    // example of how the tree path is build when current instance is on the third level of calling
+    // hierarchy:
+    // PI_<parentProcessInstanceKey>/FN_<parentCallActivityId>/FNI_<parentCallActivityInstanceKey>/
+    // PI_<secondLevelProcessInstanceKey>/FN_<secondLevelCallActivityId>/FNI_<secondLevelCallActivityInstanceKey>PI_<currentProcessInstanceKey>
```

Comment:
❓ Is the flow scope key (of the current scope) also then part of the treePath at the end? Or only the PI?

Reply:
For process instance entity no. For incident entity it is, but this is addressed in another Handler and PR.